intro-HPC/leuven/ch_hpc_policies.tex

```diff
@@ -0,0 +1,77 @@
+\chapter{HPC Policies}
+\label{ch:hpc-policies}
+
+The cluster has been setup to support the ongoing research in all the domains
+at the \university.  As such, it should be considered as
+valuable research equipment.  User shall not abuse the system for other purposes. By
+registering, users accept this implicit user agreement.
+
+In order to shared resources in a fair way, a number of site policies have been
+implemented:
+
+\begin{enumerate}
+\item  Single user node policy
+\item  Priority based scheduling
+\item  Fairshare mechanism
+\item  Crediting system
+\end{enumerate}
+
+
+\section{Single user node policy}
+
+Each user will have exclusive access to the nodes that are assigned to him.
+This means that no jobs from other users will be executed on the nodes during
+your computations.  Of course, it is very likely that multiple jobs from the
+same user are being send to the same node at the same time, in order to
+maximise the utilisation of the remaining available resources.
+
+The motivation behind this ``one node, one user'' policy is:
+
+\begin{enumerate}
+\item  Parallel jobs can suffer badly if they don't have exclusive access to the full node.
+\item  Sometimes the wrong job gets killed if a user uses more memory than requested and a node runs out of memory.  With this policy, a user cannot affected by the malicious software from other users.
+\item  A user may accidentally spawn more processes/threads and use more cores than expected (depends on the OS).
+\end{enumerate}
+
+The scheduler will (try to) fill up a node with several jobs from the same
+user. If you don't have enough work for a single node, you need a good PC and
+not a supercomputer.
+
+
+\section{Priority based scheduling?}
```

Comment:
Ok

From: Kenneth Hoste [mailto:notifications@github.com]
Sent: donderdag 18 februari 2016 14:08
To: hpcugent/vsc_user_docs <vsc_user_docs@noreply.github.com>
Cc: Mag Selwa <mag.selwa@kuleuven.be>
Subject: Re: [vsc_user_docs] Feature/leuven chapter1 (#106)


In intro-HPC/leuven/ch_hpc_policies.tex<https://github.com/hpcugent/vsc_user_docs/pull/106#discussion_r53310711>:

> +maximise the utilisation of the remaining available resources.

> +

> +The motivation behind this ``one node, one user'' policy is:

> +

> +\begin{enumerate}

> +\item  Parallel jobs can suffer badly if they don't have exclusive access to the full node.

> +\item  Sometimes the wrong job gets killed if a user uses more memory than requested and a node runs out of memory.  With this policy, a user cannot affected by the malicious software from other users.

> +\item  A user may accidentally spawn more processes/threads and use more cores than expected (depends on the OS).

> +\end{enumerate}

> +

> +The scheduler will (try to) fill up a node with several jobs from the same

> +user. If you don't have enough work for a single node, you need a good PC and

> +not a supercomputer.

> +

> +

> +\section{Priority based scheduling?}

drop the ?

—
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/vsc_user_docs/pull/106/files#r53310711>.